packages/react-icons/template.js

```diff
@@ -0,0 +1,29 @@
+function template(
+	{ template },
+	opts,
+	{ imports, interfaces, componentName, props, jsx, exports }
+) {
+	const plugins = ['jsx', 'typescript']
+	const tpl = template.smart({ plugins })
+	
+	componentName.name = componentName.name.substring(3)
+	interfaceName = 'I' + componentName.name
+	combinedExport = '{ ' + interfaceName + ', ' + componentName.name + ' }'
+	exports.declaration.name = componentName.name
+
+	return tpl.ast`
+		${imports}
+
+		interface ${interfaceName} {
```

Comment:
Rather than a different interface for each component, would it be better to have one single IFluentIconProps interface that is just used by all of them? That seems more inline with the idea that all the icons are part of a cohesive system.

Reply:
If they could use all use it, that would be ideal.

Reply:
I agree with this comment. If all the props are the same for all icons, a single shared IFluentIconProps interface would be ideal and allow for a simpler type usage downstream. Then later on if an icon has it's own props, you could have a union type, kind of like `IFluentIconProps | ISettingsIconProps`